addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -101,19 +91,13 @@ def partners(self, country=None, grade=None, page=0, **post):
         if country_all:
             url_args['country_all'] = True
 
-        partner_count = partner_obj.search_count(
-            request.cr, SUPERUSER_ID, base_partner_domain,
-            context=request.context)
+        partner_count = PartnerSudo.search_count(base_partner_domain)
```

Comment:
You are doing search with same domain on line #100 then do we need this search_count, we should write search here and we can get it using len(partners)